backend/auth.py

```
@@ -0,0 +1,72 @@
+from datetime import datetime, timedelta
+from typing import Optional
+from jose import JWTError, jwt
+import bcrypt
+from fastapi import Depends, HTTPException, status
+from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+from sqlalchemy.orm import Session
+import models
+import schemas
+from database import get_db
+
+SECRET_KEY = "your-secret-key-change-in-production"
```

Ruff (0.14.4)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)